model/Licensing/Classes/SimpleLicensingText.md

```diff
@@ -0,0 +1,26 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SimpleLicensingText
+
+## Summary
+
+A license or addition that is not listed on the SPDX License List.
+
+## Description
+
+A SimpleLicensingText represents a License or Addition that is not listed on the SPDX License
+List at https://spdx.org/licenses, and is therefore defined by an SPDX data
+creator.
+
+## Metadata
+
+- name: SimpleLicensingText
+- SubclassOf: AnyLicenseInfo
+- Instantiability: Concrete
+
+## Properties
+
+- simpleLicensingText
```

Comment:
Could we just use the `licenseText` property and move it back from the `ExpandedLicense` to `Licensing`?  It seems like it has similar semantics

Reply:
`licenseText` is used in the ExpandedLicense profile, so if we move it we would have to reference it from another profile. That would be a first, I think. I'm not sure if we currently support this. @zvr?
I intentionally renamed this so that we don't get the same property name in two different profiles (we had some discussion a while back that we would try to avoid that if possible).

Reply:
> so if we move it we would have to reference it from another profile

I think this is OK, but I'll defer to @zvr for a definitive answer on this

> I intentionally renamed this so that we don't get the same property name in two different profiles (we had some discussion a while back that we would try to avoid that if possible).

Agree we should not use the same property name in 2 different profiles - I would propose referring to the property in a different profile as long as it is supported

Reply:
We could do it (as we could do almost anything), especially since there is no way to use `ExpandedLicensing` profile without using the `SimpleLicensing` one.
But I think it would introduce complexity in tooling (and in spec-parser) to have classes in a profile having properties from other profiles. Remember, they all end up with IRIs with namespaces.

To Gary's last point "we should not use the same property name in 2 different profiles", we're already past this stage, as you all know: cf. `locator`.

Reply:
> But I think it would introduce complexity in tooling (and in spec-parser) to have classes in a profile having properties from other profiles. Remember, they all end up with IRIs with namespaces.

@zvr We already have `Software` classes referencing other classes (e.g. `SoftwareArtifact.concludedLicense` refers to a property in a namespace other than `SoftwareArtifact`).  It this situation more complex?

Since the `licenseText` and `simpleLicensingText` are semantically the same, it would be useful for the RDF users to use the same property name.  BTW - this is solveable with different property names by using `owl:sameAs`, but it would not be as straight-forward as just using the same property name.

Reply:
@goneall the example you cited will be replaced by a Relationship :-P

Do we have another?

Reply:
`/Core/PositiveIntegerRange` referenced from `Software/Snippet` is another example.

Reply:
The `licenseText` property is now moved to `SimpleLicensing`, while also still used in the `ExpandedLicensing` profile.